core/src/test/scala/com/databricks/labs/remorph/transpilers/SnowflakeToDatabricksTranspilerTest.scala

```diff
@@ -6,6 +6,19 @@ class SnowflakeToDatabricksTranspilerTest extends AnyWordSpec with TranspilerTes
 
   protected val transpiler = new SnowflakeToDatabricksTranspiler
 
+  "Snowflake non ansi command" should {
+    "transpile non ansi command" in {
+      "select * from t1,t2 where t1.x=t2.x(+) " transpilesTo
```

Comment:
```suggestion
    "select * from t1,t2 where t1.x=t2.x(+)" in {
      "select * from t1,t2 where t1.x=t2.x(+)" transpilesTo
```
follow the style of other tests

Reply:
will take care of it. Its work in progress and a draft PR, As a practice I commit the dev code on regular basis.